packages/eslint-plugin-pf-codemods/lib/rules/v5/emptyStateVariant-rename-values.js

```diff
@@ -39,6 +51,38 @@ module.exports = {
               });
             }
           },
+          JSXOpeningElement(node) {
+            if (emptyStateImport.local.name === node.name.name) {
+              const variantProp = node.attributes.find(
+                (attribute) => attribute.name?.name === "variant"
+              );
+
+              if (
+                variantProp &&
+                (rename[variantProp.value?.value] ||
+                  variantProp.value?.type === "Literal")
```

Comment:
We should be able to omit the type === Literal portion as that seems to be causing a valid test to expect an error since `variant="lg"` causes the type === Literal to be true.

```suggestion
                rename[variantProp.value?.value]
```